DiscoverSamples.ps1

```diff
@@ -1,104 +0,0 @@
-Param (    
```

Comment:
Doesn't the existing vscode launch script reference this? I don't see that file edited here. We should have a base one that still calls the right script so that there's some default behavior that sets things up with VS Code.

Reply:
This was split into 2 scripts to keep the rewrite into `./common/MultiTarget/GenerateAllProjectReferences.ps1` clean. Looks like launch.json was being ignored by git when I committed. Should be fixed now.